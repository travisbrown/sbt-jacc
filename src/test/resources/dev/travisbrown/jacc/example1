
INTEGER '+' INTEGER '*' INTEGER ';'
INTEGER '*' INTEGER '+' INTEGER

