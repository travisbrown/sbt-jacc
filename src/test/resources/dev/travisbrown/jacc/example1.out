Running example from input stream
start  :  _ INTEGER ...
shift  :  INTEGER _ '+' ...
reduce :  _ expr  '+' ...
goto   :  expr _ '+' ...
shift  :  expr '+' _ INTEGER ...
shift  :  expr '+' INTEGER _ '*' ...
reduce :  expr '+' _ expr  '*' ...
goto   :  expr '+' expr _ '*' ...
shift  :  expr '+' expr '*' _ INTEGER ...
shift  :  expr '+' expr '*' INTEGER _ ';' ...
reduce :  expr '+' expr '*' _ expr  ';' ...
goto   :  expr '+' expr '*' expr _ ';' ...
reduce :  expr '+' _ expr  ';' ...
goto   :  expr '+' expr _ ';' ...
reduce :  _ expr  ';' ...
goto   :  expr _ ';' ...
reduce :  _ prog  ';' ...
goto   :  prog _ ';' ...
shift  :  prog ';' _ INTEGER ...
shift  :  prog ';' INTEGER _ '*' ...
reduce :  prog ';' _ expr  '*' ...
goto   :  prog ';' expr _ '*' ...
shift  :  prog ';' expr '*' _ INTEGER ...
shift  :  prog ';' expr '*' INTEGER _ '+' ...
reduce :  prog ';' expr '*' _ expr  '+' ...
goto   :  prog ';' expr '*' expr _ '+' ...
reduce :  prog ';' _ expr  '+' ...
goto   :  prog ';' expr _ '+' ...
shift  :  prog ';' expr '+' _ INTEGER ...
shift  :  prog ';' expr '+' INTEGER _
reduce :  prog ';' expr '+' _ expr  $end
goto   :  prog ';' expr '+' expr _ $end
reduce :  prog ';' _ expr  $end
goto   :  prog ';' expr _ $end
reduce :  _ prog  $end
goto   :  prog _ $end
Accept!